package-lock.json

```diff
@@ -1,8 +1,13362 @@
 {
   "name": "notification-service",
   "version": "0.0.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
Uh oh, did you change the version of `npm` that you're using? What happened to cause 13k lines to be added to this file? Can we exclude changes to this file?

Reply:
Oh shoot sorry, definitely did not mean to commit package-lock changes!

Reply:
(Removed!)